spec/support/matchers.rb

```diff
@@ -48,6 +48,16 @@
       end
     end
 
+    doc.css("a[href^='https://secure.login.gov']").each do |a|
+      next if a[:lang]
+      page = a[:href]
+      link_path = URI::parse(page).path
+      next if locale == 'en'
```

Comment:
Since `locale` is known in the outer scope, should we skip/`if` this sooner than the point at which we're iterating over links?

Alternatively, should we just test English links as well?

Reply:
Yeah, I want to add tests for English as well

Reply:
Added tests for English to both internal and secure.login.gov links